coalib/bears/Bear.py

```diff
@@ -276,6 +276,8 @@ def __init__(self,
                               time limit, use 0.
         :raises TypeError:    Raised when ``message_queue`` is no queue.
         :raises RuntimeError: Raised when bear requirements are not fulfilled.
+        :param debugger       A flag which enable debugger on those bears which
+                              are specified in ``debug_bears`` settings.
```

Comment:
The docstring doesn't really apply here. Since this is working directly on a bear, it's not tied to the `debug_bears` setting in the section. The step where `debug_bears` is parsed and bears are selected to be debugged happens before. Just say simply that if `True`, a debugger is started.